tests/test_boostrap.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python3
+
+"""Test DUNE bootstrap
+
+These options are tested:
+  --create-key
+  --import-dev-key
+  --bootstrap-system-full
+"""
+
+import subprocess
+
+from common import DUNE_EXE
+
+# Globals
+NODE_NAME = "my_node"
+ACCT_NAME = "myaccount"
+
+
+
+def test_booststrap():
+
+    # Remove any existing containers.
+    subprocess.run([DUNE_EXE,"--destroy-container"], check=True)
+
+    # Start the new node.
+    subprocess.run([DUNE_EXE,"--start",NODE_NAME], check=True)
+
+    # Create an account.
+    subprocess.run([DUNE_EXE,"--create-account",ACCT_NAME], check=True)
+
+    # Create a key. Get it to a var as well.
+    public_key = None
+    private_key = None
+    stdout_result = subprocess.run([DUNE_EXE,"--create-key"], check=True, stdout=subprocess.PIPE)
```

Comment:
There is a small code duplication in those two files:
test_boostrap:[32:46]
test_keys:[24:38]

Reply:
It's possible these tests can be combined at a future date; however, I do not think the bootstrap test is mature enough as is. I believe this redundancy is acceptable.